test/integration/roles/ec2_provision_instances/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 dependencies:
   - prepare_tests
+  - setup_sshkey
   - setup_ec2
```

Comment:
`setup_ec2` still has `defaults` and `tasks`, it just no longer has `tasks/main.yml`.